web/src/views/DandisetLandingView/DandisetPublish.vue

```diff
@@ -370,6 +422,15 @@ import { draftVersion, VALIDATION_ICONS } from '@/utils/constants';
 import { RawLocation } from 'vue-router';
 import { open as openMeditor } from '@/components/Meditor/state';
 
+const PUBLISH_CHECKLIST = [
```

Comment:
This doesn't matter much, but what is the general JavaScript style policy on defining constants using all-majuscule snake case? My gut tells me this should just be `publishChecklist`.

Reply:
MDN docs say all capitals https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#basic_const_usage.

Personally I think using all-capital snake case makes it clearer that the array itself is constant, and signals that you shouldn't be doing `PUBLISH_CHECKLIST.push(...)` anywhere.